src/systems/ec/implementations/react/packages/checkbox/Checkbox.jsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import Icon from '@ecl/ec-react-component-icon/Icon';
+
+const Checkbox = ({
+  id,
+  checked,
+  disabled,
+  helperText,
+  invalid,
+  invalidIconLabel,
+  invalidText,
+  label,
+  hideLabel,
+  name,
+  value,
+  className,
+  ...props
+}) => {
+  const classNames = classnames(className, 'ecl-checkbox', {
+    'ecl-checkbox--invalid': invalid,
+  });
+
+  return (
+    <div className="ecl-form-group ecl-form-group--checkbox">
+      <div className={classNames}>
+        <input
+          {...props}
+          type="checkbox"
+          id={id}
+          defaultChecked={checked}
+          name={name || undefined}
+          className="ecl-checkbox__input"
+          disabled={disabled}
+        />
+        <span className="ecl-checkbox__icon" />
+        <span className="ecl-checkbox__icon ecl-checkbox__icon--checked">
+          <Icon shape="ui--check" size="xs" />
+        </span>
+
+        {label && (
```

Comment:
`label` is mandatory (for accessibility reasons)
the label could be hidden or displayed (default: displayed)